examples/generic/generic_systems.rs

```diff
@@ -0,0 +1,11 @@
+use bevy::ecs::component::Component;
+
+// remove all entites that aren't the player
+fn remove_entities<T: Component>(q: Query<Entity, With<T>>) {
```

Comment:
You try to use a `command` variable inside the body of the function but that does not exist yet. 
Try adding a `Commands` parameter to the function :)
```rust
fn remove_entities<T: Component>(mut commands: Commands, query: Query<Entity, With<T>>) {}
```